### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by Lioness100**
## Description

Fixed 2 random typos (this was a trial run for [a CLI I made](https://github.com/Lioness100/rspell))

## Performance impact

unknown

## Security impact

unknown

## Checklist

- [x] My code matches the project's code style and `yarn lint:fix` passes.
- [ ] I've added tests for the new feature, and `yarn test` passes.
- [ ] I have detailed the new feature in the relevant documentation.
- [ ] I have added this feature to 'Pending' in the `RELEASE_NOTES.md` file (if one exists).
- [ ] If this is a breaking change I've explained why.

### CODE REVIEW

**Review by benjie — APPROVED** [review]
Thanks 👍

**Comment by benjie** [comment]
The word wrapping will have changed in the release notes, prettier needs to run.

**Comment by Lioness100** [comment]
`yarn lint:fix` responds "Invalid option '--write' - perhaps you meant '--rule'?". Removing `--write`, `yarn lint:fix` replies "No files matching the pattern "prettier" were found.". `yarn lint` responds [error] "No files matching the pattern were found: "'**/*.{js,jsx,ts,tsx,graphql,md,json}'"". I'm on Windows.

**Comment by benjie** [comment]
Ah on windows you can’t have single quotes in shell args. Not sure about—write though, works for me and is documented. Thanks for trying.

https://prettier.io/docs/en/cli.html

Oh, again it’s the semicolon which doesn’t work for Windows.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ referencing the jobs table in a database function you may have a bad time.
 - CronItem.match can now accept either a pattern string or a matcher function
 - Jobs that were locked more than 4 hours will be reattempted as before, however
   they are slightly de-prioritised by virtue of having their `run_at` updated,
-  giving interrim jobs a chance to be executed (and lessening the impact of
+  giving interim jobs a chance to be executed (and lessening the impact of
   queue stalling through hanging tasks).
```
Comment (benjie) [inline RELEASE_NOTES.md:58]: Just a guess at what Prettier's looking for:
```suggestion
  giving interim jobs a chance to be executed (and lessening the impact of queue
  stalling through hanging tasks).
```

**Pull request merged**